mne_sandbox/preprocessing/_sns.py

```diff
@@ -0,0 +1,168 @@
+# -*- coding: utf-8 -*-
```

Comment:
git commits don't always retain the original author names if someone else moves the file around. Your call whichever way you like it.